pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+target-version = ['py39']
```

Comment:
Shouldn't we expand this to all versions that `dbbackup` supports?

Reply:
This value should effectively be considered a min value. So I'll change this to py36.